src/docs/configuration.html

```diff
@@ -28,7 +28,7 @@
 			<th scope="row">
 				<code><a href="#demo">DEMO</a></code>
 			</th>
-			<td><code>true</code> or <code>false</code></td>
+			<td><code>true</code>, <code>false</code>, or <code>static</code></td>
 			<td><code>{{defaults.DEMO}}</code></td>
 			<td>
```

Comment:
For the explanation `<td>`, should we break down the difference between `true` and `'static'` now?